bw-prep.js

```diff
@@ -0,0 +1,133 @@
+
+  const response = await prompts({
+    type: 'confirm',
+    name: 'hasDS',
+    message: 'Does your team have data science members?',
+    initial: false,
+  });
+
+  // console.log(response);
+  var packages = [
+    '@storybook/react',
+    '@storybook/addon-actions',
+    '@storybook/addon-knobs',
+    '@storybook/addon-notes',
+    '@storybook/addons',
+    '@storybook/storybook-deployer',
+    'antd',
+    'prompts',
+    'kleur',
+    'yaml',
+  ];
+  if (!response.hasDS) {
+    // remove dataviz example component
+    fs.rmdirSync('./src/components/pages/ExampleDataViz', { recursive: true });
+    // add plotly packages to be removed
+    packages.push('plotly.js', 'react-plotly.js');
+  }
+  uninstall(packages);
+  // remove lines from app index.js
+  alterIndex(response.hasDS);
+  // change ci.yml
+  removeCC();
+  // remove storybook stories
+  fs.rmdirSync('./src/stories', { recursive: true });
+  fs.rmdirSync('./.storybook', { recursive: true });
+  // remove ant files
+  fs.rmdirSync('./src/styles', { recursive: true });
+  // remove scaffold .git 😈
+  fs.rmdirSync('./.git', { recursive: true });
+  // remove this file 😱
+  fs.unlinkSync('./bw-prep.js');
+})();
+
+function removeCC() {
+  const ciFile = './.github/workflows/ci.yml';
+  const file = fs.readFileSync(ciFile, 'utf8');
+  const ci = YAML.parse(file);
+  //remove the code climate step
+  ci.jobs.coverage.steps.pop();
+  //add a build step
+  ci.jobs.coverage.steps.push({ run: 'npm run build' });
+  fs.writeFileSync(ciFile, YAML.stringify(ci), 'utf8');
+}
+
+function uninstall(packages) {
+  const packageNames = packages.join(' ');
+  console.log(kleur.bold(`uninstalling packages: ${packageNames}`));
+  exec(`npm uninstall ${packageNames}`, (error, stdout, stderr) => {
+    console.log(stderr);
+    if (error !== null) {
+      console.log(
+        kleur
+          .bold()
+          .yellow()
+          .bgRed(`exec error: ${error}`)
+      );
+    }
+    // why not run git init for them?
+    console.log(
+      kleur.bold().green('Uninstall complete'),
+      `\n\nThe scaffold is now ready for your Build Week project. You may now run ${kleur
+        .bold()
+        .underline('git init')} \n\n🎉🎉🎉🎉 🚀`
+    );
+  });
+}
+
+function alterIndex(hasDS) {
+  var indexFile = './src/index.js';
+  try {
+    var file = fs.readFileSync(indexFile, 'utf8');
+    var lines = file.split('\n');
+    replaceHomePage(lines);
+    if (!hasDS) lines = removeLines(lines, 'ExampleDataViz');
+    lines = removeLines(lines, 'antd');
+    lines = removeLines(lines, 'ProfileListPage');
+    lines = removeLines(lines, 'HomePage');
+    lines = removeLines(lines, 'LoadingComponent');
+
+    var result = lines.join('\n');
+    fs.writeFileSync(indexFile, result, 'utf8');
+  } catch (e) {
+    console.error(e);
+  }
+}
+
+function replaceHomePage(lines) {
+  // super brittle stuff here
+  lines[49] = lines[49].replace(/SecureRoute/g, 'Route');
```

Comment:
Holy cow! This will break if we change that file right? 😬

Totes fine with it, but we should document this hard core in the Readme/File itself.

Reply:
Yup, crazy scary. 

The future will need to be generator/template based rather than mutating a working app. This is just a short term hack.